cmd/juju/crossmodel/showformatter.go

```diff
@@ -37,10 +38,19 @@ func formatOfferedEndpointsTabular(writer io.Writer, all map[string]ShowOfferedA
 	tw := output.TabWriter(writer)
 	w := output.Wrapper{tw}
 
-	w.Println("URL", "Access", "Description", "Endpoint", "Interface", "Role")
+	w.Println("Store", "URL", "Access", "Description", "Endpoint", "Interface", "Role")
 
-	for name, one := range all {
-		offerName := name
+	for urlStr, one := range all {
+		url, err := crossmodel.ParseApplicationURL(urlStr)
+		if err != nil {
+			return err
+		}
+		store := url.Source
+		if store == "" {
+			store = "local"
```

Comment:
what does "local" mean in this case? local to the controller that we're querying? what other values might it take on?